tutorial/20_recipes/010_reuse_best_trial.py

```diff
@@ -92,3 +92,8 @@ def detailed_objective(trial):
 # and behaves differently from :class:`~optuna.trial.Trial` in some situations.
 # For example, pruning does not work because :class:`~optuna.trial.FrozenTrial.should_prune`
 # always returns ``False``.
+#
+# .. note::
+#     For multi-objective optimization as demonstrated by :ref:`multi_objective`,
+#     :func:`~optuna.study.Study.best_trials` returns a list of :class:`~optuna.trial.FrozenTrial`
```

Comment:
```suggestion
#     :attr:`~optuna.study.Study.best_trials` returns a list of :class:`~optuna.trial.FrozenTrial`
```

Reply:
Again, I'm so sorry. I got a bit confused and thought `Study.best_trials` as a function.